### PR TITLE
Automate census docker build

### PR DESCRIPTION
**Pull request by AdeHarrison**
# Motivation and Context
We want travis build to push docker images to Google Container Registry instead of dockerhub

# What has changed
Replaced docker image prefix with GCR registry location
NOTE: before merge, remove "automate-census-docker-build" branch condition and "branches only" section in .travis.yml file

# How to test?
From travis dashboard, switch to  "automate-census-docker-build" branch and "Restart Build". Once complete, check latest Docker image is in GCR.

# Links

[Trello](https://trello.com/c/dC6sEWqE/443-set-up-automated-docker-image-build)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,23 +19,17 @@ install: mvn install -DskipTests -DskipITs -Ddocker.skip -Ddockerfile.skip -Dmav
...
+  fi
+- bash <(curl -s https://codecov.io/bash)
 
 cache:
   directories:
   - "$HOME/.m2"
 
 branches:
   only:
-  - master
```
Comment (necrophonic) [inline .travis.yml]: This will break when it gets merged back to `rm-census`?

File: .travis.yml [hunk 2]
```diff
@@ -6,6 +6,11 @@ services:
 language: java
 jdk: openjdk8
 
+env:
+  global:
+  - SOURCE_IMAGE_NAME="samplesvc"
```
Comment (davidmort) [inline .travis.yml]: Is there any benefit the env vars given they're only used once?

File: .travis.yml (hunk 1 above)
Comment (davidmort) [reply .travis.yml]: yep, shouldn't this be "rm-census"?

File: .travis.yml [hunk 3]
```diff
@@ -14,23 +19,17 @@ install: mvn install -DskipTests -DskipITs -Ddocker.skip -Ddockerfile.skip -Dmav
...
-    fi
-  - bash <(curl -s https://codecov.io/bash)
-
-notifications:
-  slack:
-    rooms:
-      secure: V+Uvej5O5wvnrDwpD41Dj4f/EGUlVqHX3UaW99njf//gl8yCezXcnHS0G8Wkdhi2SE2ss0I2TwQpDkQzCVtR5/RL+J8oXpyOJFFiZnhCworNHQniwMigHY5eoSIQCwiKe6wJeccsdQB67XPlhI4Q0tWgh/sDvwOs3X8sptvHpTdtBBs8oVkpqMYIg/PDPTrYpbPDxiUB0eGws41eTplR7H08Lp9Gzz1rEMak4zHCuae4idJWTpcUga/LnreKpisBHnvNA7Iadyv6LRP2AP5O7gZaL27f1KL5kewk0KZndJ1qyTZv4r5CyI20V1C7ja4wV7zm11s7MFxmGHidYtKUsMx5G53kjZNsoYMgFjnbmt1LfOFiMh8Me7WN7TsmiKePGHrCDf9LQTFhzgE5OFx8dzPKMrikKlzZ0eoVKPPbKm4/WQLErBeb9MkITEsYy5ATw/Mpx2eCIuQdM+E91nz8HYVJn1Pud/duIMlQOiQKVTgvz0c2LDLCaIEK7/dzwNsVvE7IEaXOzz7BQ2P3583CP0PV9SqrnaOswcIR1QJ6w4SeIJ1FvmaFhOE/ClaUnHdq0J4KY2/HXELSSa9HlV3l73d+XY3TP9m6vjEYz+esp8zevA33CUn8D6hl7+/ds7OdxLP9AiXrTHcP1vu/bAcEBEHgqz2YRjteOdCvVtT5MLU=
-    on_failure: always
-    on_success: never
+- if ([ "$TRAVIS_BRANCH" == "rm-census" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]) || ([ "$TRAVIS_BRANCH" == "automate-census-docker-build" ]); then
```
Comment (davidmort) [inline .travis.yml]: need to remove the second part of this "if" as per the PR description before it can be merged

Comment (AdeHarrison) [reply .travis.yml]: fixed

File: .travis.yml (hunk 1 above)
Comment (necrophonic) [reply .travis.yml]: 👍

**Pull request merged**